fission-web-server/library/Fission/Web/Server/IPFS/Cluster/Client.hs

```diff
@@ -0,0 +1,19 @@
+module Fission.Web.Server.IPFS.Cluster.Client
+  ( API
+  , pinClient
+  ) where
+
+import           Fission.Prelude
+
+import           Servant
+import           Servant.Client
+
+import qualified Network.IPFS.CID.Types as IPFS
+
+type API
+  = "pins"
+  :> Capture "cid" IPFS.CID
+  :> PostNoContent
```

Comment:
🗺 The HTTP API types are down in `fission-web-api`

Reply:
I'm not sure... I went back and forth on it a bit. But decided not to. It seems a little bit strange to have only 1 Cluster method in there while the rest is just ipfs.

I could honestly got either way tho :man_shrugging:

Reply:
Maybe to clarify, this is hitting the ipfs-cluster daemon's api not the ipfs daemon

Reply:
For sure. Let's leave it like this for now. Nothing stopping us from moving it to `Network.IPFS.Cluster` down the road.

Speaking of, I feel like `ipfs-haskell` could use some TLC. It's not a priority right now, but updating some deps and cleaning up the UI now that we're more familiar with IPFS would be nice.

Reply:
yea for sure^^ especially since it got some attention recently